blocked-edges/4.12.18-DualStackNeedsController.yaml

```diff
@@ -0,0 +1,8 @@
+to: 4.12.18
+from: .*
+url: https://issues.redhat.com/browse/SDN-3996
+name: DualStackNeedsController
+message: |-
+  Upgrade can get stuck on clusters that use Whereabouts with dual stack functionality.
```

Comment:
The message is not easy to understand. Can we make it simpler? The confusion is about what `Whereabouts` means in this context.